CharacterPanelRefinedPlugin.cs

```diff
@@ -269,8 +294,13 @@ public class CharacterPanelRefinedPlugin : IDalamudPlugin {
 
         return default;
     }
+    
+    
 
     private unsafe void* CharacterStatusOnSetup(AtkUnitBase* atkUnitBase, int a2, void* a3) {
+        
+        LoadLocalization();
```

Comment:
Language should be initialized in the constructor, since it can't be changed without restarting the game

Reply:
The language of the plugin itself can be changed in the plugin options. (Optional change between ingame language, and default english)
Moving LoadLocalization to the constructer makes a plugin restart after changing the option necessary.

Reply:
Well, then the method should be called as well in the if block of the ImGui.Checkbox.